flink-python/pyflink/table/expressions.py

```diff
@@ -503,6 +503,25 @@ def map_from_arrays(key, value) -> Expression:
     return _binary_op("mapFromArrays", key, value)
 
 
+def map_from_entries(rows) -> Expression:
+    """
+    Creates a map from an array of entries (row with two fields).
+
+    Example:
+    ::
+
+        >>> tab.select(
+        >>>     map_from_entries(
+        >>>         array(row(key1, 1), row(key2, 2), row(key3, 3))
+        >>>     ))
+
+    .. note::
+
+        both arrays should have the same length.
```

Comment:
do we have 2 arrays?

Reply:
only have one arg  @snuyanzin

Reply:
then why is this sentence talking about `both` arrays?

Reply:
@snuyanzin very detail, just copy error, i fixed